packages/ui/src/components/va-tabs/hooks/useTabsService.ts

```diff
@@ -0,0 +1,51 @@
+import { ref, Ref } from 'vue'
+import { TabComponent } from './useTab'
+
+export interface TabsService {
+  tabs: Ref<TabComponent[]>,
+  disabled: Ref<boolean>,
```

Comment:
Can you check this works reactively?
I.e. if we change disabled from false to true - does it get updated in component.

Reply:
In given test cases component works fine and I didn't see any issues.